src/instructlab/configuration.py

```diff
@@ -127,6 +127,49 @@ def after_debug_level(self):
         return self
 
 
+class _rag_retriever_embedder(BaseModel):
```

Comment:
these classes are already covered by the rag_configuration module. are we going to replicate them here?
only the main `_rag` class is needed (unless we want to move it in that module as well)

Reply:
Since the new RAG options for `ilab model chat` are options alongside existing ones, it made sense to me to put them all in the same place and to follow the same pattern as the other chat options, at least for now.